testplan/base.py

```diff
@@ -449,9 +449,16 @@ class TestplanMock(Testplan):
     def __init__(self, *args, **kwargs):
         # mock testplan could run in threads
         kwargs.setdefault("abort_signals", [])
-        kwargs.setdefault("runpath", default_runpath_mock)
         kwargs.setdefault("parse_cmdline", False)
         kwargs.setdefault("logger_level", logger.DEBUG)
 
+        if os.environ.get("TEST_ROOT_RUNPATH"):
```

Comment:
Shall this if-else logic go to default_runpath_mock?

Reply:
good idea